### PR TITLE
Fix breaking bugs

### PR DESCRIPTION
**Pull request by puiterwijk**
Signed-off-by: Patrick Uiterwijk <puiterwijk@redhat.com>

### CODE REVIEW

File: tahrir/views.py [hunk 1]
```diff
@@ -1437,6 +1437,7 @@ def report_year_month(request):
     year = int(request.matchdict.get('year'))
     month = int(request.matchdict.get('month'))
 
+    now = datetime.now()
```
Comment (pypingou) [inline tahrir/views.py]: utcnow?

Comment (puiterwijk) [inline tahrir/views.py]: It is only using the date part of it, so not sure it matters... Also, our servers are all on UTC.

File: tahrir/templates/user.mak [hunk 2]
```diff
@@ -31,7 +31,7 @@
 
           % if rank != 0:
             <p>
-              Ranked ${rank} out of ${user_count} ranked users (top ${"{0:.2f}".format(percentile)}%).</p>
+              Ranked ${rank} out of ${user_count} ranked users (top ${"{:0.2f}".format(float(percentile))}%).</p>
```
Comment (sayanchowdhury) [inline tahrir/templates/user.mak]: What's the issue here?

File: tahrir/views.py (hunk 1 above)
Comment (sayanchowdhury) [inline tahrir/views.py]: +1 for this one. I thought I pushed this fix earlier

Comment (pypingou) [inline tahrir/views.py]: I'd still prefer we stick to UTC as not everyone runs their server with UTC :)

**Pull request merged**